eng/Configurations.props

```diff
@@ -20,7 +20,7 @@
   <PropertyGroup>
     <NETCoreAppCurrentVersion>5.0</NETCoreAppCurrentVersion>
     <NetCoreAppCurrentTargetFrameworkMoniker>.NETCoreApp,Version=v$(NETCoreAppCurrentVersion)</NetCoreAppCurrentTargetFrameworkMoniker>
-    <NetCoreAppCurrent>netcoreapp$(NETCoreAppCurrentVersion)</NetCoreAppCurrent>
+    <NetCoreAppCurrent>net$(NETCoreAppCurrentVersion)</NetCoreAppCurrent>
```

Comment:
Consider naming this `NetCoreAppCurrentTargetFramework` as this is what the value actually holds.

Reply:
See https://github.com/dotnet/runtime/pull/445#discussion_r353368900 for why we picked `NetCoreAppCurrent`. Also note that this property is used in thousands of places already. I wouldn't want to change its name as part of this PR.